include/libnet/libnet-functions.h

```diff
@@ -2687,7 +2689,7 @@ libnet_open_link(libnet_t *l);
  * [Internal] 
  */
 int
-libnet_close_link(libnet_t *l);
+libnet_close_link(const libnet_t *l);
```

Comment:
Forgot to add `const` here at [libnet_close_link](https://github.com/libnet/libnet/blob/39034323a4a2a366973b0be8e676cc425cf2e453/src/libnet_link_none.c#L55C1-L55C1) so that it sync with new function definiton in header that committed with this PR. Or should we remove the `const` in order to avoid breaking in future?